spacy_llm/tasks/entity_linker/util.py

```diff
@@ -206,4 +206,10 @@ def reduce_shards_to_doc(task: EntityLinkerTask, shards: Iterable[Doc]) -> Doc:
     RETURNS (Doc): Fused doc instance.
     """
     # Entities are additive, so we can just merge shards.
-    return Doc.from_docs(list(shards), ensure_whitespace=True)
+    with warnings.catch_warnings():
+        warnings.filterwarnings(
+            "ignore",
+            category=UserWarning,
+            message=".*Skipping .* while merging docs.",
+        )
+        return Doc.from_docs(list(shards), ensure_whitespace=True)
```

Comment:
Not sure where this edit is coming from?

Reply:
It's just a drive-by because I noticed the warnings filter is missing here :upside_down_face: I can move this into a separate PR, if you mind having it in here.

Reply:
It's all just a bit confusing with the huge (mostly unrelated) git history etc - I do in general appreciate more "atomic" PRs ;-)

Reply:
> I do in general appreciate more "atomic" PRs ;-)

I know 🫣

> It's all just a bit confusing with the huge (mostly unrelated) git history etc 

Yeah, I don't know why that's the case. The branches should all be updated.